### ISSUE TITLE
compile error with xcrun

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#42953]
**Godot version:**

3.2.4-beta 150f9ce807

**OS/device including version:**

not related

**Issue description:**

compile fails with iphone specific error, even though it is for x11.

**Steps to reproduce:**
```
$ scons -j 14 platform=x11 bits=64
scons: Reading SConscript files ...
Failed to find SDK version while running xcrun --sdk iphoneos --show-sdk-version.
FileNotFoundError: [Errno 2] No such file or directory: 'xcrun':
  File "godot_dev/SConstruct", line 43:
    if detect.can_build():
  File "./platform/iphone/detect.py", line 17:
    if get_darwin_sdk_version("iphone") < 13.0:
  File "godot_dev/methods.py", line 717:
    return float(decode_utf8(subprocess.check_output(["xcrun", "--sdk", sdk_name, "--show-sdk-version"]).strip()))
  File "/usr/lib/python3.8/subprocess.py", line 411:
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 489:
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.8/subprocess.py", line 854:
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702:
    raise child_exception_type(errno_num, err_msg, err_filename)
```

**Minimal reproduction project:**

**Comment by volzhs** [comment]
removing `export OSXCROSS_ROOT=path` in `~/.bashrc` fixed it.
it didn't happen before even I had it.
not sure it's a bug or fault in my env variable.

**Comment by akien-mga** [comment]
Fixed by #42955.
